### PR TITLE
Use new SBT Debian Repo before bintray is shutdown

### PR DESCRIPTION
**Pull request by Mousius**
We started seeing issues with building the TVM Docker images, and they stemmed from the SBT (Scala Build Tool) installation which was using bintray instead of a later SBT Debian URL. Thankfully bintray were just running some brown outs before they turn the service off on May 1st:
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

So I took the Scala repo URL from here:
https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html

Which is the suggested SBT URL as they may change the backend again in future:
https://github.com/sbt/sbt/issues/6294

@leandron

### CODE REVIEW

**Review by leandron — APPROVED** [review]
LGTM, can we merge this when possible @areusch ?

File: docker/install/ubuntu_install_sbt.sh
Comment (eed3si9n) [inline docker/install/ubuntu_install_sbt.sh:28]:
```suggestion
echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
```

Comment (eed3si9n) [reply docker/install/ubuntu_install_sbt.sh:28]: Sorry I haven't had time to update the docs but the new Debian packages are now under `all main`.

Comment (Mousius) [reply docker/install/ubuntu_install_sbt.sh]: Not a problem, thanks for pointing it out :smile_cat:

**Review by areusch — APPROVED** [review]
thanks @Mousius @eed3si9n ! waiting on the CI

**Comment by leandron** [comment]
I guess this is now ready to be merged?

**Comment by jcf94** [comment]
Now merged. Thanks! @Mousius @leandron @eed3si9n @areusch

**Pull request merged**
